iree/hal/cts/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ set(IREE_ALL_CTS_TESTS
   PARENT_SCOPE
 )
 
-# These tests use executables produced by the iree-translate compiler tool.
+# These tests use executables produced by the iree-compile compiler tool.
```

Comment:
per the other comment thread about `build_tools/cmake/iree_hal_cts_test_suite.cmake`, this should match what that uses (whichever tool it is actually using)